app/Models/Litter/Categories/Material.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Models\Litter\Categories;
+
+use App\Models\Litter\LitterCategory;
+
+class Material extends LitterCategory
+{
+    protected $table = 'material';
```

Comment:
Do these need to be mass assignable?

Reply:
Yep, I think we do it on the parent model.